src/zenml/zen_stores/schemas/pipeline_run_schemas.py

```diff
@@ -158,9 +158,9 @@ class PipelineRunSchema(NamedSchema, RunMetadataInterface, table=True):
             overlaps="run_metadata",
         ),
     )
-    logs: Optional["LogsSchema"] = Relationship(
+    logs: List["LogsSchema"] = Relationship(
```

Comment:
`Optional`

Reply:
The relationship between the two will change with this PR, won't it?

Reply:
Ah you're right, my bad